### PR TITLE
Hiding unreleased public api's

### PR DESCRIPTION
**Pull request by sagarchkrbrty**
Added `@hidden` tag to unreleased public `video` APIs

### CODE REVIEW

File: packages/teams-js/src/public/video.ts [hunk 1]
```diff
@@ -5,10 +5,12 @@ import { errorNotSupportedOnPlatform, FrameContexts } from './constants';
 import { runtime } from './runtime';
 
 /**
+ * @hidden
```
Comment (erinha) [inline packages/teams-js/src/public/video.ts:8]: Are these APIs still under development? If so and there are plans to continue changing them before "releasing" them, consider adding a "@beta" tag to them as well, indicating to developers that they might change and they should not build production level apps using these APIs without understanding that they might break at any time.

**Comment by AE-MS** [comment]
If these APIs are still under development, has there been a design review on this feature? I don't recall seeing it coming through office hours.
https://aka.ms/extendingteamsjssdk
Thanks!

**Comment by erinha** [comment]
You need to provide a change file by running `yarn changefile` in the root of the monorepo. Suggested change type: patch. Suggested description of your change: "Added \`@hidden\` tag to unreleased public \`video\` APIs". (Note the backtick quotes and past tense used.)

**Review by erinha — CHANGES_REQUESTED** [review]
See comments about change file and design review on APIs under development.

**Comment by sagarchkrbrty** [comment]
> If these APIs are still under development, has there been a design review on this feature? I don't recall seeing it coming through office hours. https://aka.ms/extendingteamsjssdk Thanks!

These were part of V1 SDK

**Comment by AE-MS** [comment]
> > If these APIs are still under development, has there been a design review on this feature? I don't recall seeing it coming through office hours. https://aka.ms/extendingteamsjssdk Thanks!
> 
> These were part of V1 SDK

That's good to know -- but if they are still under development, they are now being developed for the V2 SDK, correct?

File: packages/teams-js/src/public/video.ts (hunk 1 above)
Comment (sagarchkrbrty) [reply packages/teams-js/src/public/video.ts:8]: Good point. Added the tag.

**Comment by sagarchkrbrty** [comment]
> You need to provide a change file by running `yarn changefile` in the root of the monorepo. Suggested change type: patch. Suggested description of your change: "Added `@hidden` tag to unreleased public `video` APIs". (Note the backtick quotes and past tense used.)
> 
> See the [CONTRIBUTING.md](https://github.com/OfficeDev/microsoft-teams-library-js/blob/main/CONTRIBUTING.md#change-log-using-beachball) for more details.

Added the changefile

**Comment by erinha** [comment]
Since these APIs have publicly shipped (both in v1 and in v2) they need to have public documentation. We can't actually go back and remove it since we're committed to supporting these APIs now that they've been released. I suggest you attend office hours to discuss your feature and its roadmap, given that it has already been publicly released and has not been previously reviewed. https://aka.ms/extendingteamsjssdk

File: packages/teams-js/src/public/video.ts (hunk 1 above)
Comment (erinha) [reply packages/teams-js/src/public/video.ts:8]: See my other comment after further thought about already releasing these APIs.

**Comment by erinha** [comment]
Feel free to reopen after discussing the plan forward.

**Pull request closed without merging**